src/API/Middlewares/RequestValidationMiddleware.ts

```diff
@@ -0,0 +1,27 @@
+import { IllegalArgumentException } from '@po.et/poet-js'
+import * as Joi from 'joi'
```

Comment:
Can you add an empty line after the imports?

Reply:
done